.drone.yml

```diff
@@ -60,7 +60,6 @@ steps:
     commands:
       - make css
       - make js
-      - bash -c '[ -z "$(git status --porcelain public/js public/css)" ] || (echo "Generated js/css files do not match" && git status --porcelain public/js public/css && exit 1)'
```

Comment:
This step's only purpose is now to run the JS/CSS linters. I guess keeping it will allows the CI to fail fast on lint errors instead of going any further. We could extend it later to also do linting of go code (maybe https://github.com/golangci/golangci-lint).